app/src/main/java/com/kickstarter/libs/utils/extensions/AnyExt.kt

```diff
@@ -0,0 +1,54 @@
+package com.kickstarter.libs.utils.extensions
+
+import com.kickstarter.libs.utils.ObjectUtils
+import io.reactivex.functions.Function
+import rx.functions.Func1
```

Comment:
`Fun1` is rxJava1, `Function` is RXJava 2` a bit confused around the `coalesceWith` extension methods looks like they are transformers, but as they lived previously on `ObjectUtils` let's keep them for now.